Tools/clinic/clinic.py

```diff
@@ -5488,9 +5490,9 @@ def add_parameter(text):
             add(p.name)
             add('\n')
             add(textwrap.indent(rstrip_lines(p.docstring.rstrip()), "    "))
-        parameters = output()
-        if parameters:
-            parameters += '\n'
+        parameters_output = output()
```

Comment:
This was required because the local `parameters` variable previously had type `list[Parameter]` in this function, but here it's assigned to a `str`, and mypy (understandably) wasn't a fan of that

Reply:
Ahhh, sorry, didn't mean to duplicate work!

Reply:
No need to be sorry! I don't even have it as a local branch anymore. It was just an experiment :)